ansible-ipi-install/roles/installer/tasks/60_deploy_ocp.yml

```diff
@@ -7,4 +13,23 @@
 - name: Deploy OpenShift Cluster
   shell: |
     /usr/local/bin/openshift-baremetal-install --dir {{ dir }} --log-level debug create cluster
+  async: 3600
+  poll: 0
+  ignore_errors: yes
+  register: installer_result
 
+- name: Wait for Bootstrap Complete
+  shell: |
+    /usr/local/bin/openshift-baremetal-install --dir {{ dir }} --log-level debug wait-for bootstrap-complete
+  register: wait_for_bootstrap_result
+  until: wait_for_bootstrap_result is succeeded
+  retries: "{{ bootstrap_timeout|int }}"
+  delay: 1
+
+- name: Wait for Install Complete
+  shell: |
+    /usr/local/bin/openshift-baremetal-install --dir {{ dir }} --log-level debug wait-for install-complete
+  register: wait_for_install_result
+  until: wait_for_install_result is succeeded
+  retries: "{{ install_timeout|int }}"
+  delay: 1
```

Comment:
moved all this into a block that just verifies the increase_bootstrap_timeout and increase_install_timeout are defined.